README.md

```diff
@@ -112,12 +112,15 @@ Contact: firstname.lastname@hpi.de
 -	Michael  Janke
```

Comment:
(random line)
your rule would not create a predicate expression if a single node has a predicate like `PredicateNode::make(and_(greater_than_(a, 5), less_than_(a, 10))`, right? At least I haven't seen a test for it. 

Note that you can use `flatten_logical_expressions(expr, LogicalOperator::And) to untangle arbitrarily complex AND nestings.

Reply:
We added the support and tests for 'and expressions'.